packages/suite/src/components/wallet/AccountTopPanel/components/AccountNavigation/index.tsx

```diff
@@ -126,7 +137,7 @@ export const AccountNavigation = ({
             icon: 'SIGNATURE',
             position: 'secondary',
             extra: true,
-            isHidden: !account || !hasNetworkFeatures(account, 'sign-verify'),
+            isHidden: !networkFeatures?.includes('sign-verify'),
```

Comment:
this is not working as expected, I have to fix it (Sign&Verify is hidden on the first load till you click on another account)

Reply:
@komret could you please check my fixup? https://github.com/trezor/trezor-suite/pull/6367/commits/a6e693ff73aecf017bd873c1fe70e249bdc68ffb